src/js/common/schemaform/review/ReviewPage.jsx

```diff
@@ -224,10 +181,7 @@ const mapDispatchToProps = {
   submitForm,
   setPrivacyAgreement,
   setData,
-  uploadFile,
-  saveAndRedirectToReturnUrl,
-  autoSaveForm,
-  toggleLoginModal
+  uploadFile
 };
 
 ReviewPage.propTypes = {
```

Comment:
Should we add `contentAfterButtons`, `renderErrorMessage`, and `setData` in here?